Cargo.toml

```diff
@@ -130,6 +130,7 @@ unsigned-varint = { version = "0.8.0" }
 # we import via `rust-multiaddr`.
 # This is expected to stay here until we move `libp2p-identity` to a separate repository which makes the dependency relationship more obvious.
 libp2p-identity = { path = "identity" }
+yamux = { git = "https://github.com/libp2p/rust-yamux", branch = "dynamic-stream-window" }
```

Comment:
TODO: Revert before merge.

Reply:
CI tells you that too: https://github.com/libp2p/rust-libp2p/actions/runs/7104627596/job/19340030903#step:4:26847

Do note that it isn't marked as required so it won't stop merges.